data/sea/54-zebra.h

```diff
@@ -120,8 +128,8 @@ static ierror_msg_t zebra_translate
     iint_t fact_remaining = fact_count - fact_offset;
     iint_t fact_to_read   = 0;
 
-    if (fact_remaining > zebra_chunk_fact_count) {
-        fact_to_read = zebra_chunk_fact_count;
+    if (fact_remaining > state->chunk_size) {
+        fact_to_read = state->chunk_size;
```

Comment:
sorry for being pedantic but I preferred "chunk_fact_count": size makes me think of bytes or continuous things like centimetres